src/async/consumers/block.consumer.ts

```diff
@@ -131,6 +132,44 @@ export class BlockConsumer {
                                 }
                             }
                         }
+
+                        // Get Millions deposit/withdrawal information
+                        // TODO: Add deposit_update when available
+                        if (ev.type === 'deposit' || ev.type === 'withdraw_deposit') {
+                            const keyArray = ev.attributes.map((a) => a.key);
+
+                            if (keyArray.includes('pool_id') && keyArray.includes('deposit_id') && keyArray.includes('amount') && keyArray.includes('depositor')) {
+                                const id = Number(ev.attributes.find((a) => a.key === 'deposit_id').value);
+
+                                // Parse amount
+                                const amountValue = ev.attributes.find((a) => a.key === 'amount').value;
```

Comment:
What happens in case that attribute is not found? 
Trying to access value would output an undefined error

Reply:
There is a condition above that checks this